src/main/java/mekanism/common/Mekanism.java

```diff
@@ -691,6 +631,7 @@ private void registerTileEntities() {
         registerTileEntity(TileEntityElectricPump.class, "electric_pump");
         registerTileEntity(TileEntityElectrolyticSeparator.class, "electrolytic_separator");
         registerTileEntity(TileEntityEliteFactory.class, "ultimate_smelting_factory");
```

Comment:
Because the registration here is "ultimate_smelting_factory"
Changing to "elite_smelting_factory" will cause a slot ID error, which will cause a crash
[Why did the original author of Mekanism write "ultimate_smelting_factory" like this?]